### PR TITLE
Improved search module

### LINKED ISSUE #354: Showing atleast one topic
Currently, begiak with this command `.topics <any>` showing one topic in user's PM and it also shows `.more` the command to show remaining topics. How about showing at least one topic and then others topics in users PM or similar to that?

### PR DESCRIPTION
**Pull request by gstark0**
Fixes #354 and #356 :)

HTML tags are now removed (before and after fix):
screen shot 2018-01-07 at 19 24 19
screen shot 2018-01-07 at 19 23 55

### CODE REVIEW

**Comment by gstark0** [comment]
Oh, I forgot to change channel name.

**Comment by sushain97** [comment]
Build failed?

**Comment by gstark0** [comment]
@sushain97 Not related to my commit, it’s because time of timeouts. See #347

**Comment by gstark0** [comment]
Because of timeout* sorry, can’t edit on phone

File: modules/search.py [hunk 1]
```diff
@@ -61,6 +63,8 @@ def search(phenny, input):
                 return phenny.say('Sorry, no result.')
     except:
         return phenny.say('Sorry, no result.')
+    # Removes html tags, if exist
+    answer = re.sub('<[^<]+>', '', answer)
```
Comment (Androbin) [inline modules/search.py]: Shouldn't the regex be `<[^>]+>` ? Also, you can just `<.+?>` (lazy quantifier)

Comment (sushain97) [reply modules/search.py]: Yeah, this seems correct. Feel free to send a PR.

Comment (gstark0) [reply modules/search.py]: Seems okay, cool! :)

**Pull request merged**